plot.py

```diff
@@ -94,7 +94,9 @@ def create_plot(all_data, raw, x_log, y_log, xn, yn, fn_out, linestyles):
 
     dataset = get_dataset(args.dataset)
     dimension = len(dataset['train'][0]) # TODO(erikbern): ugly
-    point_type = 'float' # TODO(erikbern): should look at the type of X_train
+    point_type = 'float'
+    if 'point_type' in dataset.attrs:
+        point_type = dataset.attrs['point_type']
```

Comment:
i think you can just do `dataset.attrs.get('point_type', 'float')`

Reply:
👍 Done